llvm/unittests/IR/ConstantsTest.cpp

```diff
@@ -835,5 +835,33 @@ TEST(ConstantsTest, BlockAddressCAPITest) {
   EXPECT_EQ(&BB, OutBB);
 }
 
+TEST(ConstantsTest, Float128Test) {
+  LLVMTypeRef Ty128 = LLVMFP128TypeInContext(LLVMGetGlobalContext());
```

Comment:
Please do not use `LLVMGetGlobalContext` here, which is about to be deprecated. Use LLVMContextCreate + LLVMContextDispose instead.